scene/3d/scene_importer_mesh_node_3d.h

```diff
@@ -31,8 +31,8 @@
 #ifndef EDITOR_SCENE_IMPORTER_MESH_NODE_3D_H
```

Comment:
I propse a different design for this
* Keep this editor side only
* Give GLTFDocument a function pointer or class to generate the mesh and mesh node, so editor uses this version
* If the above is not provided, GLTFDocument simply generates ArrayMesh and MeshInstance3D

Reply:
Huh, I remember you did this for SurfaceTool. That's a good idea.